locust/templates/index.html

```diff
@@ -57,6 +57,13 @@
             <div class="padder">
                 <h2>Start new Locust swarm</h2>
                 <form action="/swarm" method="POST" id="swarm_form">
+                    <label for="locust_class">Locust to run</label>
```

Comment:
I wonder if it would be nice for this to be a checkbox list or something that let you pick multiples but not all?